tests/auth_tests/test_hashers.py

```diff
@@ -596,3 +596,22 @@ def setter(password):
             self.assertTrue(state['upgraded'])
         finally:
             setattr(hasher, attr, old_value)
+
+
+@override_settings(PASSWORD_HASHERS=PASSWORD_HASHERS)
```

Comment:
This won't work anymore without `ScryptPasswordHasher` in `PASSWORD_HASHERS`.

Reply:
Removed in most recent commit.